code/game/objects/effects/effect_system.dm

```diff
@@ -595,7 +595,7 @@ steam.start() -- spawns the effect
 		spawn(0)
 			var/turf/T = get_turf(src.holder)
 			if(T != src.oldposition)
-				if(istype(T, /turf/space))
+				if(istype(T, /turf/space) || istype(T, /turf/simulated/open))
```

Comment:
It would PROBABLY be better to have this property of being "empty" be a var of some sort but I guess this is fine for now

Reply:
Could make it a macro, maybe